cognite/client/_api/iam.py

```diff
@@ -187,17 +187,49 @@ def __init__(self, config: ClientConfig, api_version: str | None, cognite_client
         super().__init__(config, api_version, cognite_client)
         self._LIST_LIMIT = 100
 
-    def create(self, client_credentials: ClientCredentials | None = None) -> CreatedSession:
+    def __call__(self, client_credentials: ClientCredentials | dict | None = None) -> CDFSession:
```

Comment:
This can also be implemented using the @contextmanager decorator, and a try:finally around the `yield`. Personally, I like that syntax better. But this works too!

Reply:
➕1 vote in favor of the decorator 🎉

Reply:
Completely forgot about it, changed the implementation now.